src/plugins/saved_objects_management/public/management_section/mount_section.tsx

```diff
@@ -15,12 +15,10 @@ import { EuiLoadingSpinner } from '@elastic/eui';
 import { CoreSetup } from 'src/core/public';
 import { ManagementAppMountParams } from '../../../management/public';
 import { StartDependencies, SavedObjectsManagementPluginStart } from '../plugin';
-import { ISavedObjectsManagementServiceRegistry } from '../services';
 import { getAllowedTypes } from './../lib';
 
 interface MountParams {
   core: CoreSetup<StartDependencies, SavedObjectsManagementPluginStart>;
-  serviceRegistry: ISavedObjectsManagementServiceRegistry;
```

Comment:
We use the raw saved objects directly now and don't need the client-side registry.